clang/test/Driver/sycl-native-cpu.cpp

```diff
@@ -17,3 +17,11 @@
 // CHECK-WIN: {{.*}}"-fsycl-is-device"{{.*}}"-gcodeview"
 // CHECK-WIN-DAG: {{.*}}"-fsycl-is-host"{{.*}}"-gcodeview"
 // CHECK-WIN-NOT: dwarf
+
+// checks that -sycl-opt is not enabled by default on NativeCPU so that the full llvm optimization is enabled
+// RUN:   %clang -fsycl -fsycl-targets=native_cpu -### %s 2>&1 | FileCheck -check-prefix=CHECK-OPTS %s
+// CHECK-OPTS-NOT: -sycl-opt
+
+// checking that explicitly enabling -sycl-opt still works
+// RUN:   %clang -fsycl -fsycl-targets=native_cpu -mllvm -sycl-opt -### %s 2>&1 | FileCheck -check-prefix=CHECK-OPT_ON %s
+// CHECK-OPT_ON: -sycl-opt
```

Comment:
This check doesn't make sense to me. We already have tests checking testing `-mllvm` option and I don't think this test adds any value.

Reply:
I removed the check. Thank you.